### ISSUE TITLE
Installer

### ISSUE THREAD

**Issue by Nicola1971** [fourroses666/bootstrap-template#1]
The template is very nice, basic, but not "install friendly"
You need to convert the package to evo installer format (Package Manager and Store modules compatible).

I will help you, if you want :)

**Comment by Nicola1971** [comment]
OK, i've done most of the job :)
The installer works fine and install a complete and working template with all wayfinder tpls.
I've created some "example" chunks too (see the example tpl file for updates)
It's a very simple "cut and paste" routine.. If you help me , we can complete the package very soon.

**Comment by fourroses666** [comment]
Hey, thanks :) 

It is/was not installable indeed, didn't know howto, glad you fixed that. 
Actually hoped it would be default on Evo, the old one is way to old but as installer its fine too with me!

Its still "far" from perfect.
We can do this, but I must say its way to busy at the moment, got back from vacation, got sick and now just started to work again and projects won't give me airtime.

So if you can hold on, I probably will have a look next week :(

**Comment by Nicola1971** [comment]
I agree with you about the old evo template, but if you want your boostrap theme as default template, you need to make it compatible with evo installer.

About the template:
 Pros:
* very clean 
* Nice palette
* well coded (love the order in your styles)

Cons:
* Not user friendly: you can't feel the power of modx+boostrap
* Too much empty: some of your example chunks should be integrated in the template (ie:slider)

Some examples:

* My Start-Evo Template : it's still a beta, requires too much 3d party snippets and plugins for a default template, but you can check a lot of using of evo features+boostrap
https://github.com/Nicola1971/START-Evo

* Default Clipper CMS bootstrap template : simple, but powerfull
https://github.com/ClipperCMS/ClipperCMS

**Comment by fourroses666** [comment]
Yea Installer is good. 
I still need to look what you have done already, after that I check for improvements.
Need to install it on a clean Evo. Yea, we can intergrade a slider. 

ps. its bootstrap not boostrap :P

**Comment by Nicola1971** [comment]
I'm a bit dyslexic :P
